src/libraries/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -128,7 +129,7 @@ internal static void HtmlEncode(string? value, TextWriter output)
         private static int IndexOfHtmlAttributeEncodingChars(string s) =>
             s.AsSpan().IndexOfAny("<\"'&");
 
-        private static bool IsNonAsciiByte(byte b) => b >= 0x7F || b < 0x20;
+        private static bool IsNonAsciiOrSpaceByte(byte b) => (uint)b - 0x20 - 1 >= 0x7F - 0x20 - 1;
```

Comment:
```suggestion
        private static bool IsNonAsciiOrSpaceByte(byte b) => char.IsBetween((char)b, (char)0x20, (char)0x7F);
```
I think this is better for readability

Reply:
Actually since this is only used in 1 place, you can just delete this helper and do the `char.IsBetween` there